Client/Frontend/Browser/BrowserViewController/BrowserViewController+Menu.swift

```diff
@@ -211,6 +125,81 @@ extension BrowserViewController {
       }
     }
   }
+  
+  private func navigateToBraveNews() {
```

Comment:
@mauriciomaniglia can you remove the functions for the actions and add all the logic directly inside the completion blocks here? We are not using these function more than once and it makes it irrelevant and hard to track this way.

Reply:
Yes. But before I do that, are you sure about that? I moved the actions to separate functions to make the code more readable, look: 

![Screen Shot 2022-09-27 at 17 50 13](https://user-images.githubusercontent.com/7022519/192634587-abbb63ba-1d48-4273-87fc-50b11900ca92.png)

Even though those functions have not been used more than once, when you separate the code into blocks and glue those functions altogether, it looks nicer to read. 

If we keep all logic inside the completions blocks, it will look like this:

![Screen Shot 2022-09-27 at 17 59 05](https://user-images.githubusercontent.com/7022519/192635418-7750c56e-2bc0-4db9-8423-3fd352b37d5c.png)
![Screen Shot 2022-09-27 at 17 59 24](https://user-images.githubusercontent.com/7022519/192635434-0186bc43-8cc9-4d37-93d6-0d6c2cb43a42.png)

For me is fine anyway, is just a matter of taste 🙂

Reply:
Yeap pretty sure

Reply:
Done ✅